spec/cassettes/update_shipment_contents/update/failure.yml

```diff
@@ -0,0 +1,61 @@
+---
+http_interactions:
+- request:
+    method: post
+    uri: https://apistaging.newgisticsfulfillment.com/update_shipment_contents.aspx
+    body:
+      encoding: ASCII-8BIT
+      string: |
+        <?xml version="1.0"?>
+        <Shipment apiKey="<API_KEY>" id="INVALID">
+          <Items>
+            <Item>
+              <SKU>1007-201-G</SKU>
+              <Qty>1</Qty>
+            </Item>
+          </Items>
+          <AddItems>#&lt;Nokogiri::XML::Builder::NodeBuilder:0x007fb60b18f1c8&gt;</AddItems>
```

Comment:
This seems weird, why is it logging the `Builder` object as part of the XML?

Reply:
this was an error on my part in the code, thanks for catching!